app/controllers/events_controller.rb

```diff
@@ -225,6 +225,12 @@ def ledger
       @mock_total = @transactions.sum(&:amount_cents)
     end
 
+    @compact = @organizer_position.compact_mode
+    if params[:compact].present?
+      @compact = params[:compact] == "true"
+      @organizer_position.update(compact_mode: @compact)
+    end
```

Comment:
Imo this shouldn't be configurable - and we should pick one design and stick with it